lib/ES/Repo.pm

```diff
@@ -5,49 +5,25 @@ use warnings;
 use v5.10;
 
 use Path::Class();
-use URI();
 use Encode qw(decode_utf8);
 use ES::Util qw(run sha_for);
 
+use base qw( ES::BaseRepo );
```

Comment:
It would be better to use a new [parent](https://metacpan.org/pod/parent) directive (more lightweight) instead of the old-school `base`.

Reply:
`parent` look great! I'll switch to it